packages/interface/src/utils/pcd.ts

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable */
```

Comment:
are none of these exported by any of zupass libraries?

Reply:
Just checked their codebase and seems like they use some of this code internally, not sure why they don't export it,  will ask around why, if not will submit a PR to them